spring-boot-postgresql/accelerator.yaml

```diff
@@ -0,0 +1,166 @@
+            - text: "postgres-database"
+              with: "#databaseResourceName"
+        - type: OpenRewriteRecipe
+          recipe: org.openrewrite.yaml.MergeYaml
+          options:
+            key: "'$.spec'"
+            yaml: "'serviceClaims:\n  - name: postgres-db\n    ref:\n      apiVersion: sql.tanzu.vmware.com/v1\n      kind: Postgres\n      name: ' + #databaseName"
+
+    - include: [ "config/postgres.yaml" ]
+      chain:
+        - type: ReplaceText
+          substitutions:
+            - text: "postgres-database"
+              with: "#databaseResourceName"
+        - type: ReplaceText
+          substitutions:
+            - text: "postgresdatabase"
+              with: "#databaseName"
+
+    - include: [ "pom.xml" ]
+      chain:
+        - type: ReplaceText
+          regex:
+            pattern: "(?<startOfDependencies><dependencies>)(?<existingDependencies>(?![\\s\\S]+<groupId>org.postgresql</groupId>\\s*<artifactId>postgresql</artifactId>[\\s\\S]+</dependencies>))"
+            with: |
+              '${startOfDependencies}
+                      <dependency>
+                          <groupId>org.postgresql</groupId>
+                          <artifactId>postgresql</artifactId>
+                          <scope>runtime</scope>
+                      </dependency>${existingDependencies}'
+        - type: ReplaceText
+          regex:
+            pattern: "(?<startOfDependencies><dependencies>)(?<existingDependencies>(?![\\s\\S]+<groupId>org.testcontainers</groupId>\\s*<artifactId>postgresql</artifactId>[\\s\\S]+</dependencies>))"
+            with: |
+              '${startOfDependencies}
+                      <dependency>
+                          <groupId>org.testcontainers</groupId>
+                          <artifactId>postgresql</artifactId>
+                          <version>1.17.3</version>
+                          <scope>test</scope>
+                      </dependency>${existingDependencies}'
+
+    - include: [ "build.gradle.kts" ]
+      chain:
+        - type: ReplaceText
+          regex:
+            pattern: "(?<startOfDependencies>dependencies \\{)(?<existingDependencies>(?![\\s\\S]+implementation\\s*\\(\"org.postgresql:postgresql[\\s\\S]+))"
+            with: |
+              '${startOfDependencies}
+                  implementation("org.postgresql:postgresql")${existingDependencies}'
+        - type: ReplaceText
+          regex:
+            pattern: "(?<startOfDependencies>dependencies \\{)(?<existingDependencies>(?![\\s\\S]+testImplementation\\s*\\(\"org.testcontainers:postgresql[\\s\\S]+))"
+            with: |
+              '${startOfDependencies}
+                  implementation("org.testcontainers:postgresql:1.17.3")${existingDependencies}'
+
+    - include: [ "README.md" ]
+      chain:
+        - merge:
+            - type: ReplaceText # No-op to trick the engine into thinking there IS a merge
+          onConflict: FavorForeign
+        - type: ReplaceText
```

Comment:
So you didn't extract that in a file after all?

Reply:
No, we didn't.